src/components/React/VolumeRenderer.js

```diff
@@ -97,6 +96,8 @@ export default connect(mapStateToProps)(
                     z_spacing: state.z_spacing,
                   }}
                   position="0 0 0"
+                  rotation="-55 0 0"
```

Comment:
these initial values should be moved into the config/state - @RobertGemmaJr can you address this when you merge in and connect everything up? (same with the arcball-camera below)

Reply:
Do you think should be a separate PR? Right now my branch doesn't touch the "backend" aframe stuff

Reply:
wherever/whenever makes the most sense.  It seems like it'll be easier once everything is `Context`, so it can be during or as a follow on to that.